src/components/JuzMarker/markNumbers.js

```diff
@@ -0,0 +1,155 @@
+export const juzStart = {
```

Comment:
Where do these come from? Can you write a note about this?

Reply:
Okay i'll add a note and a link to the source

Reply:
Or we use api to get juz for any ayah, like we discussed on slack. https://quran.api-docs.io/v3/juz/get-all-juzs

Reply:
@naveed-ahmad  I feel like its abit of an over kill calling an api for every ayat to check whether it 's a start of a juz
thoughts?

Reply:
Hmm... feels like this should be a bootstrap API call. Might be an overkill for this PR imho since this will have little cost to size for the time being until we make it into an API